receiver/sqlqueryreceiver/config.go

```diff
@@ -14,6 +14,8 @@ import (
 
 type Config struct {
 	sqlquery.Config `mapstructure:",squash"`
+	// The maximumn number of open connections to the sql server. <= 0 means unlimited
+	MaxOpenConn int `mapstructure:"max_open_conn"`
```

Comment:
Please also add to README. Please indicate default value in README.

Reply:
Added in README. Thanks